### ISSUE TITLE
Russian time zones were changed on Oct 26, 2014

### ISSUE THREAD

**Issue by peterdenk** [bcit-ci/codeigniter3-translations#69]
Three Russian time zones were changed on Oct 26, 2014 and in https://github.com/bcit-ci/CodeIgniter/pull/3444 date_lang.php was updated accordingly. Now it also needs to be updated in each language.

Line 73: $lang['UP3'] = '(UTC +3:00) **Moscow Time**, East Africa Time, Arabia Standard Time';
Line 75: $lang['UP4'] = '(UTC +4:00) Azerbaijan Standard Time, **Samara Time**';
Line 89: $lang['UP11'] = '(UTC +11:00) **Srednekolymsk Time**, Solomon Islands, Vanuatu';

**Comment by jim-parry** [comment]
Good point! Each of the maintainers should step up, I hope!

**Comment by appleboy** [comment]
@peterdenk :+1:

**Comment by jim-parry** [comment]
I will shortly be tagging the "master" branch for release, and adding a "develop" branch for future changes. As part of the housekeeping to prepare for this, I have asked my UCOSP student team to propose fixes to this issue, for all the translations that have not done so already!

**Comment by jim-parry** [comment]
@vadimyer We took a look at the Russian date_lang, and could not figure what to propose in the way of changes to address this issue. Can you take a look, please?
Thanks!

**Comment by ivantcholakov** [comment]
date_lang: A backup proposal from a non-Russian:

```
line 73: $lang['UP3'] = '(UTC +3:00) Московское время, Восточноафриканское время, Ирак, Кувейт, Саудовская Аравия';
line 75: $lang['UP4'] = '(UTC +4:00) Самарское время, ОАЭ, Оман, Азербайджан, Армения, Грузия';
line 89: $lang['UP11'] = '(UTC +11:00) Среднеколымское время, Центрально-тихоокеанское время (Соломоновы Острова, Новая Каледония)';
```

**Comment by vadimyer** [comment]
@jim-parry @ivantcholakov is right on all 3 lines.

I would also modify these for Russian date_lang:
$lang['UP2'] = '(UTC +2:00) Калининградское время, Восточноевропейское время, Египет, Израиль, Ливан, Ливия, Турция, ЮАР';
$lang['UP12'] = '(UTC +12:00) Камчатское время, Маршалловы Острова, Фиджи, Новая Зеландия';

 This is the latest correct information from Wikipedia: http://en.wikipedia.org/wiki/Time_in_Russia.

**Comment by ivantcholakov** [comment]
@vadimyer I think, you need to make a pull-request.

**Comment by vadimyer** [comment]
I've made it, but I don't know how to "attach" it to this thread, just curious.

**Comment by ivantcholakov** [comment]
At your repository, the main page, up-left, a green button with a tip "Compare, review, create a pull-request.".

**Comment by vadimyer** [comment]
Yep the request has been created, I mean could you "mention" it in this thread like people above?

**Comment by ivantcholakov** [comment]
Sorry, I can see you did the pull request. Ok.

**Comment by jim-parry** [comment]
I have had the UCOSP team submit suggested translations using Google translate, in order to trigger activity on the part of the translation maintainers. We need to get the repo up to snuff, or individual translations will not make it into the "3.0" release!

Any translations which do not have a response by midnight PST on Monday Jan 19th will have the proposed Google translations merged! The repo will be tagged as 3.0 then, and split into "master-3.0" and "develop" (the new default) branches.

**Comment by jim-parry** [comment]
All translations have been updated in preparation for version 3,0.
Those maintainers who did not respond in a timely fashion will have to submit pull requests for any corrections.
